src/plugins/infoaction/structure_plugin.js

```diff
@@ -0,0 +1,85 @@
+/* structure_plugin.js (c) 2014 by Michael Hausl [michael at hausl dot com]
```

Comment:
Can you explain a bit more what this plugin will do? (E.g. in the forum as I guess others might want to use it as well)